docs/changelog/assets/css/app.css

```diff
@@ -0,0 +1,241 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    font-family: 'Roboto', sans-serif;
+    background-color: #f5f5f5;
+    margin: 0;
+    display: flex;
+    flex-direction: column;
+    min-height: 100vh;
+}
+
+.header_anchor{
+    text-decoration: none;
+}
+
+#header {
+    position: sticky;
+    top: 0;
+    background-color: #00bceb; /* Webex blue */
+    color: white;
+    padding: 20px;
+    margin: 0;
+    text-align: left; /* Align the title text to the left */
+    box-shadow: 0 2px 5px rgba(0, 0, 0, 0.2);
+    font-size: 1.5rem;
```

Comment:
For box shadow, px is fine because it won't affect smaller screens.

For font size, we kept rem so that on smaller screens, the size is still not too much